Gemfile.lock

```diff
@@ -15,16 +22,49 @@ GEM
       safe_yaml (~> 1.0.0)
     daemons (1.2.3)
     eventmachine (1.0.9.1)
+    faraday (0.9.2)
+      multipart-post (>= 1.2, < 3)
+    google-api-client (0.9.9)
+      addressable (~> 2.3)
+      googleauth (~> 0.5)
+      httpclient (~> 2.7)
+      hurley (~> 0.1)
+      memoist (~> 0.11)
+      mime-types (>= 1.6)
+      representable (~> 2.3.0)
+      retriable (~> 2.0)
+      thor (~> 0.19)
+    googleauth (0.5.1)
+      faraday (~> 0.9)
+      jwt (~> 1.4)
+      logging (~> 2.0)
+      memoist (~> 0.12)
+      multi_json (~> 1.11)
+      os (~> 0.9)
+      signet (~> 0.7)
     hashdiff (0.3.0)
+    httpclient (2.8.0)
+    hurley (0.2)
     i18n (0.7.0)
     json (1.8.3)
+    jwt (1.5.4)
+    little-plugger (1.1.4)
+    logging (2.1.0)
+      little-plugger (~> 1.1)
+      multi_json (~> 1.10)
+    memoist (0.14.0)
+    mime-types (3.1)
+      mime-types-data (~> 3.2015)
+    mime-types-data (3.2016.0521)
     minitest (5.9.0)
     minitest-reporters (1.1.7)
       ansi
       builder
       minitest (>= 5.0)
       ruby-progressbar
     multi_json (1.12.1)
+    multipart-post (2.0.0)
+    os (0.9.6)
```

Comment:
Just curious, is `os` an add-on or is it part of a typical Ruby base installation?